docs/how-to/contribute-code.rst

```diff
@@ -22,7 +22,10 @@ coding:
 
     $ python3 -m venv venv
     $ source venv/bin/activate.sh
-    (venv) $ git clone https://github.com/pybee/rubicon-objc.git
+    (venv) $ pip install --upgrade pip
+    (venv) $ pip install --upgrade setuptools
+    (venv) $ pip install tox
+    (venv) $ git clone https://github.com/beeware/rubicon-objc.git
     (venv) $ cd rubicon-objc
     (venv) $ pip install -e .
```

Comment:
These instructions first create the virtual environment before cloning the repo, meaning that the env will be located outside of the rubicon-objc repo directory. Is that intentional? The usual convention I know is to put the env into the directory of the project that it belongs to, not next to it. (This isn't always the right location of course - sometimes you want to put the env elsewhere because you want to use it with multiple projects for example, but I don't think that needs to be explained in the "newcomer development setup" instructions.)

Reply:
Fair point; I've adjusted the order.